### DISCUSSION TITLE
SQL foreign multi levels...

### DISCUSSION THREAD

**Discussion by on-acscrz** [supabase/supabase#6672] [Questions]
Hello,

i`m from Brazil, i have the question...

I created three tables:

- table1
- table2
fk_table_1
- table3
fk_table 2

and... i`m trying to create SQL:
Select * from table 1 
left join table2 table1_id = table1_id 
left join table3 table2_id = table2_id

how i create a SQL for Supabase?

**Comment by steve-chavez** [comment]
See https://supabase.com/docs/reference/javascript/select#query-foreign-tables. Otherwise create a SQL function and call it through [rpc](https://supabase.com/docs/reference/javascript/rpc).
